pages/AddSurveyQuestionCommonUtils.js

```diff
@@ -0,0 +1,58 @@
+    await expect(page.getByRole('button', { name: 'Cancel' })).toBeEnabled();
+    await expect(page.getByRole('button', { name: 'Reset' })).toBeEnabled();
+    await page.getByRole('button', { name: 'Reset' }).click();
+  }
+
+export async function questionFieldsCharacterLimit(page){
+  await page.getByPlaceholder('Enter question code').fill('Q11111111111111111111111111111111111111111111111111111111111112');
+  await expect(page.locator("(//p[normalize-space()='63/64'])")).toBeVisible();
+  await page.keyboard.down('Control');
+  await page.keyboard.press('Z');
+  await page.keyboard.up('Control');
+  await page.getByPlaceholder('Enter question', { exact: true }).fill('Tell more about the company Tell more about the company Tell more about the company Tell more about the company Tell more about the company Tell more about the company Tell more about the company Tell more about the company Tell more about the company Tell more about the company Tell more about the company Tell more about the company Tell more about the company Tell more about the company Tell more about the company Tell more about the company Tell more about the company Tell more about the company Tell mo');
+  await expect(page.locator("(//p[normalize-space()='511/512'])")).toBeVisible();
+}
+
+export async function optionFieldsCharacterlimit(page){
+  await page.getByPlaceholder('Enter option').first().fill('Not at all willingTell more abut the company Tell more about the company Tell more about the company Tell more about the company Tell more about the company Tell more about the company Tell more about the company Tell more about the company Tell more about the company Tell more about the company Tell more about the company Tell more about the company Tell more about the company Tell more about the company Tell more about the company Tell more about the company Tell more about the company Tell more about th');
+  await expect(page.locator("(//p[normalize-space()='511/512'])").nth(1)).toBeVisible();
+  await page.getByPlaceholder('Enter code').first().fill('Not at all willingTell more abou the compan Tell more about the');
+  await expect(page.locator("(//p[normalize-space()='63/64'])")).toBeVisible();
+}
+export async function entryFieldsCharacterLimit(page){
+  await page.getByPlaceholder('Enter title for the field').first().fill('Field Entry 1 Fi1eld Entry 1Field Entry 1Field Entry 1Field Entry 1Field Entry 1Field Entry 1Field Entry 1Field Entry 1Field Ent');
+  await expect(page.locator("(//p[normalize-space()='128/128'])")).toBeVisible();
+  await page.getByPlaceholder('Enter hint for the field').first().fill('Field Entry 1 Field Entry 1Field Entry 1Field Entry 1Field Entry 1Field Entry 1Field Entry 1Field Entry 1Field Entry 1Field Entr');
+  await expect(page.locator("(//p[normalize-space()='128/128'])").nth(1)).toBeVisible();
```

Comment:
(//p[normalize-space()='128/128'])" is repeated here. Is there a way we can define all these things in 1 place with proper name and then use it?